docs/knn/settings.md

```diff
@@ -64,3 +64,56 @@ Statistic |  Description
 `script_compilation_errors` | The number of errors during script compilation.
 `script_query_requests` | The number of query requests that use [the KNN script](../#custom-scoring).
 `script_query_errors` | The number of errors during script queries.
+
+## Tasks
```

Comment:
Unfortunately, I think this is just the wrong spot for this content. The _tasks API isn't specific to KNN; any content on it should just go into the Elasticsearch section. That said, documenting the entire _tasks API is a large *task* in and of itself, so I think the right call here is to just include the simple GET call in the other file, like this:

"
To monitor the warmup operation, call the Elasticsearch _tasks API:

```
GET _tasks
```
"

You can omit the response, revert the header changes, and we'll (you'll?) eventually document the _tasks API fully under the Elasticsearch header.